pkg/protodep/protodep.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
```

Comment:
I have a use case for this in a project which doesn't currently require solo-kit. It would be nice to have this in go-utils or even a standalone repo

Reply:
Planning on doing that once it's all in and working, as well as adding support for git repos